eralchemy/main.py

```diff
@@ -232,6 +276,9 @@ def _intermediary_to_dot(tables, relationships, title=""):
     "er": intermediary_to_markdown,
     "md": intermediary_to_mermaid,
     "dot": intermediary_to_dot,
+    "puml": intermediary_to_puml,
+    "uml": intermediary_to_puml,
+    "pu": intermediary_to_puml,
```

Comment:
I don't like having something as generic as uml here.
puml and pu seems fine.
In the input mode, I don't think we need more than one option, so I would remove pu as well?